docs/getting-started/quickstart.md

```diff
@@ -39,49 +43,69 @@ The [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line
 
-Developers select a specific blockchain and build a Substreams module tailored to the selected chain.&#x20;
+Before continuing, ensure that your system [meets the basic requirements](https://substreams.streamingfast.io/developers-guide/installation-requirements) for Substreams development.
 
-Data is available for any blockchain exposing an operational Firehose endpoint, installed and set up in [an on-premises environment](https://firehose.streamingfast.io/firehose-setup/ethereum/installation-1), or [provided by StreamingFast](../reference-and-specs/chains-and-endpoints.md) or other vendors.
+### Objectives
 
-{% hint style="info" %}
-**Note**: The remaining documentation assumes the [`substreams` CLI](../reference-and-specs/command-line-interface.md) and all other required dependencies have been installed and a [StreamingFast authentication token](../reference-and-specs/authentication.md) has been obtained.&#x20;
-{% endhint %}
+- Create your first Substreams module
+- Use the [`substreams` CLI](https://substreams.streamingfast.io/reference-and-specs/command-line-interface) to run the module
 
-### **Basics**
+### Prerequisites
 
-The most basic approach to use Substreams is through the [`substreams` CLI](../reference-and-specs/command-line-interface.md), passing an endpoint and the name of the Rust function, or module handler, used by the compute engine for processing.&#x20;
+- Go compiler
+- Rust compiler
+- [`substreams` CLI](https://substreams.streamingfast.io/reference-and-specs/command-line-interface)
+- StreamingFast authentication key
 
-The manifest defines the name of the module handler and the protocol buffer to use as the data definition for the Substreams module.&#x20;
+### 1. Create Substreams manifest
 
-The commands demonstrate how Substreams modules work by using the selected blockchain and instructing the compute engine to execute the `map_basic_eth` or `map_basic_sol` Rust functions, also called "_module handlers_".
+To create a "Substreams module", you must first create the manifest file. Use the example manifest provided in the [`substreams-ethereum-tutorial`](https://github.com/streamingfast/substreams-ethereum-tutorial)repository on GitHub as a guide. This example manifest includes the minimal required fields to demonstrate the core values that you must provide.
 
-Use the [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) command for the Ethereum example by using:
+To use the example manifest, copy and paste it into a new file named substreams.yaml. Save this file in the root directory of your "Substreams module." You can find the example manifest in the [official GitHub repository for `substreams-ethereum-tutorial`](https://github.com/streamingfast/substreams-ethereum-tutorial).
```

Comment:
https://github.com/streamingfast/substreams-ethereum-tutorial is reference twice. I would remove:

> Use the example manifest provided in the [`substreams-ethereum-tutorial`](https://github.com/streamingfast/substreams-ethereum-tutorial)repository on GitHub as a guide.

From the first paragraph. Then, both can be merged together in one.

Also, shouldn't `substreams.yaml` be in monospace so wrapped with backticks?

Reply:
Is this a typo `... of your "Substreams module."` ? The ending dot within the `"`

Reply:
Sentence removed. Content merged. No, that was correctly formatted, but I went ahead and removed the quotes for the second reference to Substreams module. The punctuation goes inside the quotes at the end of sentences "like this."